concurrency-primer.tex

```diff
@@ -939,7 +939,7 @@ \subsection{ABA problem}
 At the end, the dangling pointer could either point to garbage or trigger segmentation fault.
 It could be even worse if nested ABA problem occurs in thread B.
 Also, the possibility to allocate a job with same address could be higher when using memory pool, meaning that more chances to have ABA problem occurred.
-In fact, pre-allocated memory should be used to achive lock-free since \monobox{malloc} could have mutex involved in multithreaded environment.
+In fact, pre-allocated memory should be used to achieve lock-free since \monobox{malloc} could have mutex involved in multithreaded environment.
```

Comment:
Always use "multi-threaded" instead of "multithreaded."

Reply:
Resolved in 8abe58d.